AudioManagerUtilities/test/AmSocketHandlerTest/CAmSocketHandlerTest.h

```diff
@@ -188,6 +236,20 @@ namespace am
         void SetUp();
         void TearDown();
     };
+    
+    class CAmSamplePluginStressTest: public CAmSamplePlugin
+    {
+        std::vector<CAmTimerStressTest2*> mTimers;
+    public:
+        CAmSamplePluginStressTest(CAmSocketHandler *mySocketHandler, sockType_e socketType);
+        ~CAmSamplePluginStressTest();
+        
+        virtual void receiveData(const pollfd pollfd, const sh_pollHandle_t handle, void* userData);
+        virtual bool dispatchData(const sh_pollHandle_t handle, void* userData);
+        virtual bool check(const sh_pollHandle_t handle, void* userData);
```

Comment:
In case it is the final implementation of your unit test virtual shouldn't be used but instead use the specifier *final*:
void foobar(... , ...) final;